src/js/nav-secondary.js

```diff
@@ -7,7 +7,7 @@
   if (navigation) {
     const toggle = navigation.querySelector('[data-nav-secondary-toggle]');
     const navigation_sub = navigation.querySelector('[data-nav-secondary-sub]');
-    const media_query = window.matchMedia('(width < 64em)');
+    const media_query = window.matchMedia('(max-width: 64em)');
```

Comment:
Nice 👍 

Nitpick: technicallllly, as we're using `max-height` should this be 1023px (63.9375em). I.e. this behaviour goes away at 1024 (64em) 🤔

Reply:
Aha, good thinking.

I'll add the same to the `off-canvas.js` too.